src/components/sdk/DeviceSelection/CameraSelection/VideoFilterSelection.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { ChangeEvent, useEffect, useState } from 'react';
+
+import { FormField } from '../../../ui/FormField';
+import { Select } from '../../../ui/Select';
+import { useBackgroundBlur } from '../../../../providers/BackgroundBlurProvider';
+import { useMeetingManager } from '../../../../providers/MeetingProvider';
+import { isVideoTransformDevice } from 'amazon-chime-sdk-js';
+
+interface Props {
+  /** Label shown for video filter selection, by default it is "Video Filter" */
+  label?: string;
+  /** Label shown in the dropdown when no video filter has been selected yet, by default it is "Select video filter" */
+  labelForUnselected?: string;
+}
+
+export const VideoFilterSelection: React.FC<Props> = ({
+  label = 'Video Filter',
+  labelForUnselected = 'Select video filter',
+}) => {
+  const {isBackgroundBlurSupported, createBackgroundBlurDevice} = useBackgroundBlur();
+  const [filterName, setFilterName] = useState('unselected');
+  const meetingManager = useMeetingManager();
+  const filterOptions = [
```

Comment:
Another reason this may be a good demo component is localization of these filter options label, we may want to ask builders to provide props so they can customize it. Still the value will be english if we are to handle it in library else we have to provide guidance on what values to provide for builders to improve on localization here.

Reply:
This is just a solution for background blur only. Its not a generic component - we may add that in the future.

Reply:
Sorry, but localization issue still stands, If builders want to provide label in their own language should no it be a prop like the `labelForUnselected` ?

Reply:
This code is no longer there right? We changed it to checkbox now.